README.md

```diff
@@ -34,7 +34,7 @@ After cloning the project, we will create the local database that the project wi
 - `npm run recreate-db:local` (this will create and populate your new team's DB)
 
 ### Database schema 
-![img](docs/Database_schema.png)
+![img](docs/Database-schema.png)
```

Comment:
Is that PR just changing the filename to kebab-case? Not that it matters but lot but I tend to follow a specific name convention for the file names. Usually in JS for components and the rest of the files we are using `camelCase` but for static files like that, you can name them using `kebab-case` as we usually do when naming files/folders in the OS for easier navigation.